gatsby/pages.rst

```diff
@@ -16,64 +23,60 @@ If we see the file structure of our `hello-world` project, we can see that there
     │       └── page-2.js
     ...
 
-Let's see how is made `index.js` file. This page is the homepage of our example site.
+Let us see how is made ``index.js`` file.
 
-.. code-block:: none
+This page is the home page of our example site.
 
-  import React from 'react'
-  import { Link } from 'gatsby'  
+.. literalinclude:: _snippets/index_orig.js
+  :language: jsx
 
-  const IndexPage = () => (
-    <div>
-      <h1>Gatsby Site</h1>
-      <p>Welcome to your new Gatsby site.</p>
-      <p>Now go build something great.</p>
-      <Link to="/page-2/">Go to page 2</Link>
-    </div>
-  )  
-  export default IndexPage
+As we said previously, pages are not basic HTML documents, but they are React components.
+You can see that React components are written in a particular syntax called ``JSX``.
+``JSX`` allows us to mix pure JavaScript with some HTML tags.
+Components are functions (or ES6 classes) that accept some data and renders some HTML.
 
-As we said previously, pages are not basic html documents, but they are React components.
-You can see that React components are written in a particular syntax called `JSX` that allows to mix pure javascript with
-some html tags.
-Components are functions (or es6 classes) that accept some data and renders some html.
+.. note::
 
-.. note:: You can see more infos in the official `ReactJS documentation <https://reactjs.org/docs/components-and-props.html>`_
+    You can see more information in the official `ReactJS documentation <https://reactjs.org/docs/components-and-props.HTML>`_
 
 
 Exercise
 ++++++++
 
-Try to edit `index.js` file and see how the homepage will change.
+Try to edit ``index.js`` file and see how the home page will change.
 
-.. note:: remember that with `gatsby develop` command, there is a webpack dev-server running with hot reload,
-so every time we make some changes, the page will automatically update.
-
-..  admonition:: Solution
-    :class: toggle
+.. note::
 
-    .. code-block:: none
+    Remember that with ``gatsby develop`` command, there is a webpack dev-server running with hot reload.
+    Every time we make some changes, the page will automatically update.
 
-      <div>
-        <h1>Hi Plone people</h1>
-        <p>Welcome to your new Gatsby site.</p>
-        <p>Now go build something great.</p>
-        <Link to="/page-2/">Go to page 2</Link>
-      </div>
+..admonition:: Solution
```

Comment:
It needs exactly 1 space, not 2 and not 0, between the `..` and `admonition`.

`.. admonition:: Solution`

See https://github.com/plone/training/pull/341#discussion_r228763123